Neos.Flow/composer.json

```diff
@@ -37,10 +37,10 @@
 
         "ramsey/uuid": "^3.0",
```

Comment:
Just noticed, that should be `"^3.0 || ^4.0"` to match the collection composer.json, no?

Reply:
Yes, obviuosly someone changed only the autogenerated collection manifest in https://github.com/neos/flow-development-collection/commit/99a6c74b193d810ea6274ab51726cc71db48cc13 😎

Reply:
Ah, that was a depdendabot PR.